doc/rest-api.md

```diff
@@ -739,7 +739,7 @@ Input (simple rename):
         "name": "new-name"
     }
 
-Input (migration across lxd instances):
+Input (migration across lxd instances or lxd cluster members):
```

Comment:
s/node/member/

Reply:
Nothing to amend here afaics, as it's already "member".